pkg/kyverno/test/test_command.go

```diff
@@ -343,23 +353,22 @@ func testCommandExecute(dirPath []string, valuesFile string, fileName string, gi
 	return rc, nil
 }
 
-func getLocalDirTestFiles(fs billy.Filesystem, path, fileName, valuesFile string, rc *resultCounts) []error {
+func getLocalDirTestFiles(fs billy.Filesystem, path, fileName, valuesFile string, rc *resultCounts, count *int, dirTestYamlNameCount *int) []error {
 	var errors []error
-	var count int
-	var dirTestYamlNameCount int
+
 	files, err := ioutil.ReadDir(path)
 	if err != nil {
 		return []error{fmt.Errorf("failed to read %v: %v", path, err.Error())}
 	}
 	for _, file := range files {
 		if file.IsDir() {
-			getLocalDirTestFiles(fs, filepath.Join(path, file.Name()), fileName, valuesFile, rc)
+			getLocalDirTestFiles(fs, filepath.Join(path, file.Name()), fileName, valuesFile, rc, count, dirTestYamlNameCount)
 			continue
 		}
 		if strings.Contains(file.Name(), fileName) || strings.Contains(file.Name(), "test.yaml") {
-			count++
+			*count++
 			if strings.Contains(file.Name(), "test.yaml") {
```

Comment:
If we are searching for deprecated files which are named as **test.yaml** shouldn't this become - 
```strings.Compare(file.Name(),"test.yaml") == 0``` instead of Contains?